app/Http/Controllers/TopicController.php

```diff
@@ -24,16 +25,30 @@ class TopicController extends ApiController
      */
     private function getMetaDataForModel(Topic $topic)
     {
+        if (!empty($topic->likes()->where('user_id', Auth::user()->id)->get()->first())) {
+            $is_user = true;
+            $like_id = $topic->likes()->where('user_id', Auth::user()->id)->get()->first()->id;
+            $countOfLikes=$topic->likes()->count();
```

Comment:
Please use camel case for variables.
$countOfLikes - ok.
$is_user - not ok.
$like_id - not ok.

add spaces before and after sign '='